src/tex/appendix.tex

```diff
@@ -447,8 +447,19 @@ \subsection{Impact of sampling temperature}
 \end{figure}
 
 
+\clearpage
+\subsection{Summary of trends and recommendations}
+
+\oone consistently leads across both \ChemBench corpus and \chembenchmini.  \ClaudeThreeFiveSonnet, \GPTFourO, \LlamaThreeOneFourZeroFiveBInstruct form the next tier of strong performers. These models show robust performance across all skills and difficulty levels. 
+
+Larger models generally perform better (\LlamaThreeOneFourZeroFiveBInstruct > \LlamaThreeOneSeventyBInstruct > \LlamaThreeOneEightBInstruct). However, smaller but well-designed models can compete with larger models. For example, \GemmaTwoNineBIt has an overall accuracy of 0.48 on \ChemBench corpus, which is only 10\% less accurate than the much larger \LlamaThreeOneFourZeroFiveBInstruct model.
```

Comment:
you might call it overkill, but it might be nice to make these number a `\variable` to automatically update if we change the corpus

Reply:
this is done now, can you have a look?

Reply:
there is also a conflict oops